### PR TITLE
cleanup(build): remove traceur-based Dart transpiler

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

**Comment by alexeagle** [comment]
FYI @mprobst

File: tools/transpiler/spec/types_spec.js [hunk 1]
```diff
@@ -71,10 +71,7 @@ export function main() {
 
         expect(function() {
           wf.name = true;
-        }).toThrowError(IS_DARTIUM ?
-          // Dart
-          "type 'bool' is not a subtype of type 'String' of 'value'" :
-          // JavaScript
+        }).toThrowError(
```
Comment (mprobst) [inline tools/transpiler/spec/types_spec.js]: Why does this file survive?

Comment (alexeagle) [inline tools/transpiler/spec/types_spec.js]: It's removed in a later snapshot...

Comment (mprobst) [inline tools/transpiler/spec/types_spec.js]: Shouldn't we be able to just kill tools/transpiler wholesale at some point?

Comment (alexeagle) [inline tools/transpiler/spec/types_spec.js]: Yes, I think so, but right now it still has the other "AtScript" patches we've applied to traceur for other purposes in writing the cjs and es6 output. When we stop using traceur to transpile "atscript" then I think we can complete this.

**Comment by alexeagle** [comment]
Merged in 4c1e978536031bb7cc8cb646f3a0ea180e1e99ce

**Pull request closed without merging**